src/js/views/forms/form/form-history_views.js

```diff
@@ -0,0 +1,94 @@
+import hbs from 'handlebars-inline-precompile';
+import { View } from 'marionette';
+
+import 'sass/modules/buttons.scss';
+
+import Droplist from 'js/components/droplist';
+
+import PreloadRegion from 'js/regions/preload_region';
+
+const HistoryDroplist = Droplist.extend({
+  popWidth: 224,
+  viewOptions: {
+    className: 'button-filter form__button-response',
+    template: hbs`<span class="u-margin--r-4">{{far "history"}}</span> {{formatDateTime _created_at "AT_TIME"}}{{far "angle-down"}}`,
```

Comment:
I'm hoping this icon doesn't need the u-margin class

Reply:
How should we handle the case where there are two icons inside a filter button? In the `button-filter` case it makes sense to target the `angle-down` icon specifically because that will be a consistent icon on filter dropdown menus. But the icon the left could be anything (in this case the history icon). So do we target those individually as we need them since we can't just target `.icon`? That doesn't seem very scalable.